packages/react-juce/src/components/Slider.tsx

```diff
@@ -242,22 +248,14 @@ export class Slider extends Component<PropsWithChildren<SliderProps | any>, Slid
   _onMouseDrag(e: SyntheticMouseEvent) {
     let value = 0.0;
 
-    if (this.props.hasOwnProperty('mapDragGestureToValue') && typeof this.props.mapDragGestureToValue === 'function') {
+    if (typeof this.props.mapDragGestureToValue === 'function') {
```

Comment:
I wonder if we should flip this around:

```
if (typeof this.props.mapDragGesture !== 'function') {
  console.warn('You passed a weird mapping function! Abort!');
  return;
}
```

And then just carry on as usual below?

Reply:
Ah yeah good shout. We could adopt that across the board for non-native props like this.

Reply:
Do you think log or use `invariant` ?

Reply:
Hm good point, let's use `invariant`